activities/supporting-codebase-governance/customized-governance-workshop/index.md

```diff
@@ -0,0 +1,45 @@
+---
+type: Index
+---
+
+# Customized governance workshop
+
+A customized governance workshop can be used for a codebase that already has some form of governance in place, or is considering a draft proposal.
+The state of the governance may suggest what to focus the workshop on, but it's strength is leveraging the governance game to socialize understanding of the governance.
+As such, it can be used to simply sketch out a currently well functioning mechanism, talk through an initial draft or proposed changes, or used to surface tensions, pain points, and differing understandings.
+It may also be used to talk through different scenarios to stress test the governance mechanism.
+
+## The basics
+
+The [general guidelines of how to run the governance game](../game/run-governance-game-workshop.md) can be applied with a few exceptions.
+
+### Starting state
+
+Instead of starting from a state card, ask the participants to visualize how (the draft for) governance currently works, using the available cards to describe the actors that are involved, and what relationships they have.
+This step may be quite revealing, so let this take time as it's a good opportunity for the participants to explain how they percieve status quo.
+If you can and it makes sense, prepare custom cards for the specific actors in the codebase as it will be easier for participants to relate to them.
+
+### Scenario
+
+The scenario cards are not needed, instead the real situation is used.
+If you can get someone in the community to summarize the current situation in the codebase, perhaps why this workshop was called, it can serve as the scenario setting.
```

Comment:
```suggestion
If you can get someone in the community to summarize the current situation in the codebase, including why this workshop was necessary, it can serve as the scenario setting.
```